<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeographicViewConfigWidget</class>
 <widget class="QWidget" name="GeographicViewConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-11</y>
        <width>372</width>
        <height>641</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Use shared property for:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QCheckBox" name="shapeCheckBox">
           <property name="text">
            <string>Shape</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="layoutCheckBox">
           <property name="text">
            <string>Layout</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="sizeCheckBox">
           <property name="text">
            <string>Size</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Polygon mode shape file:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="useDefaultShape">
         <property name="text">
          <string>&amp;Use default shape</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QRadioButton" name="useCsvFile">
           <property name="text">
            <string>Use csv f&amp;ile</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If you want to import polygons from a csv file &lt;br/&gt;into this view, it must be in the format :&lt;br/&gt;id lng lat&lt;br/&gt;...&lt;br/&gt;with id : id of the polygon&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="textFormat">
            <enum>Qt::RichText</enum>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <property name="indent">
            <number>24</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QLineEdit" name="csvFile">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="csvFilePushButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QRadioButton" name="usePolyFile">
           <property name="text">
            <string>Use po&amp;ly file</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;a &lt;a href=&quot;https://wiki.openstreetmap.org/wiki/Osmosis/Polygon_Filter_File_Format#Premade_polygon_files&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;poly file&lt;/span&gt;&lt;/a&gt; defines polygons to use as map background.&lt;/p&gt;&lt;p&gt;You can download .poly files on &lt;br/&gt;&lt;a href=&quot;https://download.geofabrik.de/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;https://download.geofabrik.de/&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="textFormat">
            <enum>Qt::RichText</enum>
           </property>
           <property name="indent">
            <number>24</number>
           </property>
           <property name="openExternalLinks">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QLineEdit" name="polyFile">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="polyFilePushButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="mapToPolygon">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Map to polygon</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Leaflet Custom Tile Layer url:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter a tile layer url in the form&lt;br/&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;http[s]://[{s}.]&amp;lt;domain&amp;gt;/&amp;lt;path&amp;gt;/{z}/{x}/{y}[{r}][.png]&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="customTileLayerUrl">
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string>https://{s}.basemaps.cartocdn.com/rastertiles/voyager/{z}/{x}/{y}{r}.png</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>useCsvFile</sender>
   <signal>toggled(bool)</signal>
   <receiver>csvFile</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>usePolyFile</sender>
   <signal>toggled(bool)</signal>
   <receiver>polyFile</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useCsvFile</sender>
   <signal>toggled(bool)</signal>
   <receiver>csvFilePushButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>usePolyFile</sender>
   <signal>toggled(bool)</signal>
   <receiver>polyFilePushButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>csvFilePushButton</sender>
   <signal>clicked()</signal>
   <receiver>GeographicViewConfigWidget</receiver>
   <slot>openCsvFileBrowser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>polyFilePushButton</sender>
   <signal>clicked()</signal>
   <receiver>GeographicViewConfigWidget</receiver>
   <slot>openPolyFileBrowser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mapToPolygon</sender>
   <signal>clicked()</signal>
   <receiver>GeographicViewConfigWidget</receiver>
   <slot>mapToPolygonSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openPolyFileBrowser()</slot>
  <slot>openCsvFileBrowser()</slot>
  <slot>openCsvHelp()</slot>
  <slot>openPolyHelp()</slot>
  <slot>mapToPolygonSlot()</slot>
 </slots>
</ui>
